.dotnet/src/Custom/Images/ImageClient.cs

```diff
@@ -141,13 +124,13 @@ public ImageClient(string model, ApiKeyCredential credential = default, OpenAICl
 
         options ??= new();
 
-        using MultipartFormDataBinaryContent content = options.ToMultipartContent(fileStream, fileName, prompt, _clientConnector.Model, imageCount);
+        using MultipartFormDataBinaryContent content = options.ToMultipartContent(fileStream, fileName, prompt, _clientConnector.Model!, imageCount);
```

Comment:
Curious on this one -- how do we know `_clientConnector.Model` will never be null?  If it won't ever be null, should its type be non-nullable?